Emux.GameBoy/Cpu/InterruptVector.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Emux.GameBoy.Cpu {
```

Comment:
Please put brace on next line to conform with the style of the rest of the project